Frontend/Gmfctn/src/app/shared/components/say-thank-modal/say-thank-modal.component.ts

```diff
@@ -80,8 +80,10 @@ export class SayThankModalComponent implements OnInit {
 
   private calculateBadges(): void {
     this.achList.forEach((element) => {
+
```

Comment:
Don't think this empty lines are required here

Reply:
Fixed